chirho/explainable/handlers/explanation.py

```diff
@@ -61,7 +61,7 @@ def SearchForExplanation(
     *,
     alternatives: Optional[Mapping[str, Intervention[S]]] = None,
     factors: Optional[Mapping[str, Callable[[T], torch.Tensor]]] = None,
-    preemptions: Optional[Mapping[str, Union[Intervention[S], Intervention[T]]]] = None,
+    # preemptions: Optional[Mapping[str, Union[Intervention[S], Intervention[T]]]] = None,
```

Comment:
Can you revert the change deleting this argument? I don't think it's correct and in any case it's not related to the other change that swaps the orders of the witness and antecedent handlers.